client/src/api/index.js

```diff
@@ -109,10 +109,13 @@ export const userGetORCID = (authCode, originUrl) => {
   return credentialResponse
 }
 
-export const userSubmitGrantComplaint = async (message) => {
-  // TODO: hook this up the the relevant API endpoint
-  console.log('This message will be sent to the grants team: ', message)
-  return {}
+export const userSubmitGrantComplaint = async (message, authorization) => {
```

Comment:
this should be a top level attribute so it can be accessed like `api.issue.create(issue, token)`  where issue is whatever the endpoint takes

Reply:
Just made this change!